_resources/port1.0/group/python-1.0.tcl

```diff
@@ -306,14 +351,19 @@ proc python_get_defaults {var} {
             }
         }
         binary_suffix {
+            # TODO: pypy
             if {[string match py-* [option name]]} {
-                return -${python.branch}
+                if {[string match py* ${python.version}]} {
+                    return -pypy${python.branch}
+                } else {
+                    return -${python.branch}
+                }
             } else {
                 return ""
             }
         }
         jobs_arg {
-            if {${python.version} >= 35 && [option use_parallel_build]} {
+            if {${python.branch} >= 3.5 && [option use_parallel_build]} {
```

Comment:
Once again will break with 3.10. Needs to use vercmp if python.version can no longer be used due to being non-numeric.